File: BackEnd/src/main/java/edu/icet/ecom/dto/Order.java
package edu.icet.ecom.dto;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

import java.util.ArrayList;
import java.util.List;

@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
public class Order {
    private Integer id;
    private String date;
    private Integer employeeId;
    private String employeeName;
    private Integer customerId;
    private Double total;
    private String paymentType;
    private List<OrderDetail> orderDetails= new ArrayList<>();
}

File: FrontEnd/src/app/models/Order.ts
export class Order{
    id: number|null;
    date: string;
    employeeId: number;
    employeeName: string;
    customerId: number;
    total: number;
    paymentType: string

    constructor(id: number|null, date: string, employeeId: number, employeeName: string, customerId: number, total: number, paymentType: string) {
        this.id = id;
        this.date = date;
        this.employeeId = employeeId;
        this.employeeName = employeeName;
        this.customerId = customerId;
        this.total = total;
        this.paymentType = paymentType;
    }
}